server/scripts/emails.ts

```diff
@@ -41,8 +47,83 @@ export const createNotificationEmailObject = (notification_data: IPostNotificati
     to: null,
     from: 'Commonwealth <no-reply@commonwealth.im>',
     subject: subjectLine,
+    templateId: 'd-7ccab17c309f45dab93b4290b3d396d1',
     text: `${subjectLine}. <a href='${path}'>Click here</a>`,
     html: `${subjectLine}. <a href='${path}'>Click here</a>`,
   };
   return msg;
 };
+
+// regular notification email service
+
+export const createRegularNotificationEmailObject = async (user, notifications) => {
+  let emailObjArray = [];
+  emailObjArray = await Promise.all(notifications.map(async (n) => {
+    const { created_at, root_id, root_title, root_type, comment_id, comment_text,
+      chain_id, community_id, author_address, author_chain } = JSON.parse(n.notification_data);
+    const decodedTitle = decodeURIComponent(root_title).trim();
+    const nSubscription = await n.getSubscription();
+    const { category_id } = nSubscription;
+    const content = (category_id === NotificationCategories.NewComment) ? `New comment on '${decodedTitle}'`
```

Comment:
Can we break this into if/else?

Reply:
Currently it's consistent with how we render notifications client-side.
I can, if you still want, break it into if/else statements